dockerprobe/dockerprobe.go

```diff
@@ -21,7 +23,7 @@ func (p DockerProbe) Name() string {
 	return p.name
 }
 
-func (p DockerProbe) Check() error {
+func (p DockerProbe) Check(ctx context.Context) error {
```

Comment:
If ctx is not used the variable should be named `_` like that: `Check(_ context.Context)`

Reply:
Ack, will change this where it's needed